Makefile

```diff
@@ -53,10 +53,12 @@ flow:
 	./node_modules/.bin/flow check --strip-root
 
 lint:
-	./node_modules/.bin/eslint scripts $(SOURCES) *.js --format=codeframe --rulesdir="./scripts/eslint_rules"
+	./node_modules/.bin/eslint scripts $(SOURCES) '*.js' '**/.*.js' --format=codeframe --rulesdir="./scripts/eslint_rules"
 
 fix:
-	./node_modules/.bin/eslint scripts $(SOURCES) *.js --format=codeframe --fix --rulesdir="./scripts/eslint_rules"
+    # The config is hardcoded because otherwise prettier searches for it and also picks up some broken package.json files from tests
```

Comment:
I'm surprised this worked 🤔

I assume make is parsing it as a comment at the end of the dependency list.

Makefiles must only ever be indented with tabs to avoid confusion...

Reply:
Oh didn't know that. will fix it

Reply:
Yeah, tabs are actually significant in Makefiles; very annoying :(